docs/node/migrations/zero-downtime-migrations/tutorial.mdx

```diff
@@ -0,0 +1,158 @@
+
+```bash
+kwil-admin migrate status 4b0d1ad8-f9a5-5252-97e6-387d47b1ee9e --rpcserver /tmp/kwild_0.socket
+```
+```bash
+jsonrpc.Error: code = -400, message = failed to retrieve resolution, data = null
+err code = -400, msg = failed to retrieve resolution
+```
+However `kwil-admin node status` will show the migration status of the node as shown below:
+
+```bash
+kwil-admin node status --rpcserver /tmp/kwild_0.socket
+```
+```bash
+"migration": {
+    "status": "MigrationNotStarted",
+    "start_height": 143,
+    "end_height": 2143
+}
+```
+
+## Genesis State
+
+Once the proposal is approved, the network is set to start the migration process after the specified activation period. The snapshots and genesis files generated at the start of the migration can be downloaded using the `kwil-admin migrate genesis-state` command as shown below.
+
+```bash
+kwil-admin migrate genesis-state --root-dir /tmp/chain2  --rpcserver /tmp/kwild_1.socket
+```
+
+Note that the command will not download any files if there is no approved migration or the migration has not yet started. Please retry the command after the migration is activated.
+
+```bash
+No genesis state to download yet. Migration is set to start at block height: 143
+```
+
+:::warning
+The `genesis.json` and `snapshot.sql.gz` files are generated in the directory specified using the `--root-dir` flag. Ensure that you don't run the command with `--root-dir` flag set to the current node's root directory as it can overwrite the existing genesis file in the node's root directory.
+
+If the admin service is accessible from the new node it can be run on the new node directly.
+:::
+
+```bash
+kwil-admin migrate genesis-state --root-dir /tmp/chain2  --rpcserver /tmp/kwild_1.socket
+Migration State:
+        Start Height: 143
+        End Height: 2143
+        Downloaded Genesis File: .testnet2/genesis.json
+        Downloaded Snapshot File: .testnet2/snapshot.sql.gz
+```
+:::info
+If the node is still catching up, the genesis file won't be generated until it has fully synced with the network.
+:::
+
+
+## Start New Network
+The new network should be configured to bootstrap from the snapshot file provided by the old network and replicate the state changes from the old network during the migration period. 
+
+1.  Install the new versions of the `kwild` binaries. Please refer to the [installation guide](/docs/daemon/installation) for more details.
+2. [Download the snapshot and genesis files](#genesis-state) generated at the start of the migration as shown above.
+    The genesis file downloaded will have the following information set:
+    - `chain_id`: Chain id of the new network.
+    - `app_hash`: This field is set based on the hash of the downloaded snapshot file at the migration start height. The data app hash field is mandatory to ensure that all the nodes in the network start with the same state using the snapshot files.
```

Comment:
For `app_hash`, `start_height`, and `end_height`: ... "This should not be changed.

For `chain_id` and `validators`:  This may be modified as long as the modifications are used for all nodes on the new network.

As with regular / offline migrations, it is necessary to make the genesis.json and genesis_state files available for all nodes.  One or more trusted sources should be made available to distribute these.